### PR TITLE
feat: Order alphabetically list of local authorities (FPLA-2188)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
LAs
https://tools.hmcts.net/jira/browse/FPLA-2188

### CODE REVIEW

**Review by nkoranne — COMMENTED** [review]
Just for my reference : does all future LAs needs to be inserted manually based on the alphabetical order?  and also do we need to change DisplayOrder every time when we insert any LA in between?

**Comment by tomaszpowroznik** [comment]
> Just for my reference : does all future LAs needs to be inserted manually based on the alphabetical order? and also do we need to change DisplayOrder every time when we insert any LA in between?

yes, should be added manually in correct place. Yes reindexing will be required, unless we want to do gaps in numbering (e.g 5)

**Review by nkoranne — DISMISSED** [review]

**Review by joez-f — APPROVED** [review]

**Review by nkoranne — APPROVED** [review]

File: ccd-definition/FixedLists/CareSupervision/AuthorityFixedList.json [hunk 1]
```diff
@@ -2,115 +2,134 @@
...
+    "ListElementCode": "ERY",
+    "ListElement": "East Riding of Yorkshire Council",
+    "DisplayOrder": 10
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "STF",
-    "ListElement": "Staffordshire County Council"
+    "ListElementCode": "FPLA",
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/CareSupervision/AuthorityFixedList.json:19]: I know that it was already in there but should this be here, does it describe us or is it for something else? If it is us would it not make more sense for it to be in a -nonprod file.

Comment (tomaszpowroznik) [reply ccd-definition/FixedLists/CareSupervision/AuthorityFixedList.json:19]: @lewisbirks , confirmed with Adam, this is legacy org used long time ago to do smoke testing in prod. Currently not used. Cannot submit case for such org, see fpl:
  local_authority_codes_forbidden_case_submission: FPLA

**Pull request merged**